bin/tag-release-continue.js

```diff
@@ -24,11 +25,17 @@ const callback = async options => {
 	if (options.branch.includes("promote-release")) {
 		flow = filterFlowBasedOnDevelopBranch(options, promoteContinue);
 
+		if (options.scripts[`post${options.command}`]) {
```

Comment:
Doing a `tag-release continue` does not work for me. I forced a conflict state when I was trying `tag-release promote`.

My guess is the `options.command` might be `continue` (hence looking for a `postcontinue` script and not the previous command that needed to be continued.